rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3506,7 +3510,6 @@ static rmw_ret_t rmw_take_ser_int(
       if (message_info) {
         message_info_from_sample_info(info, message_info);
       }
-
```

Comment:
Nit: unnecessary whitespace change

Reply:
Fixed in https://github.com/ros2/rmw_cyclonedds/pull/485/commits/2b33c0bea1e4045d5f8c86ee2184b16f297c6b99.